services/led.md

```diff
@@ -1,23 +1,13 @@
-# LED
+# RGB LED
 
-    identifier: 0x1fb57453
+    identifier: 0x1e3048f8
     camel: led
 
-A controller for 1 or more LEDs connected in parallel.
+A controller for 1 or more RGB LEDs connected in parallel.
 
 ## Animation steps
 
-Animations are described using pairs of intensity and duration.
-For example, the following animation
-`(0, 10ms), (0.5, 5ms), (0.5, 10ms), (1, 5ms), (0, 7ms), (0, 0ms)`
-will gradually rise the intensity `0 - 0.5` in 10ms,
-then, it will keep it steady for at `0.5` for 5ms,
-then it will rise it again `0.5 - 1` over 10ms,
-drop `1 - 0` in 5ms,
-and keep it at 0 for 7ms more.
-Any entry with duration of 0ms is considered to be end-marker.
-
-To get steady glow at `x`, use animation of `(x, 60000ms), (x, 0ms)` and keep `max_iterations` at `0xffff`.
+Animations are described using pairs of HSV and duration, similarly to the LED service.
 
 ## Registers
```

Comment:
it could have `max_power` register like the pixel led service

Reply:
yup it's there.